### PR TITLE
SAW - Add Inactive Indicator

### PR DESCRIPTION
**Pull request by SamWord**

### CODE REVIEW

File: app/assets/javascripts/dashboard/protocols.js.coffee [hunk 1]
```diff
@@ -83,9 +83,12 @@ $(document).ready ->
         if data["filterrific[with_status][]"].length
           data["filterrific[with_status][]"] = $("#filterrific_with_status").val()
 
-        if data["filterrific[with_organization][]"].length
+        if data["filterrific[with_organization][]"] && data["filterrific[with_organization][]"].length
```
Comment (wtholt) [inline app/assets/javascripts/dashboard/protocols.js.coffee]: Wasn't this already done in PR - #620?

File: app/views/dashboard/protocol_filters/_filter_protocols_form.html.haml [hunk 2]
```diff
@@ -50,7 +50,11 @@
         .form-group.row
           = form.label :with_organization, t(:dashboard)[:protocol_filters][:organization], class: 'col-lg-3 control-label'
           .col-lg-9.drop-underneath
-            = form.select :with_organization, filterrific.select_options[:with_organization], {}, class: "form-control selectpicker core-select", multiple: true, data: { none_selected_text: t(:constants)[:prompts][:any] }
+            %select.selectpicker.form-control.core-select#filterrific_with_organization{ multiple: true, name: "filterrific[with_organization][]", data: { none_selected_text: t(:constants)[:prompts][:any] } }
```
Comment (amcates) [inline app/views/dashboard/protocol_filters/_filter_protocols_form.html.haml]: = form.select :with_organization, nil, {}, class: "form-control selectpicker core-select", multiple: true, data: { none_selected_text: t(:constants)[:prompts][:any] } do

File: app/views/dashboard/protocol_filters/_filter_protocols_form.html.haml [hunk 3]
```diff
@@ -50,7 +50,11 @@
...
+              - filterrific.select_options[:with_organization].each do |org_type|
+                %optgroup{ label: org_type[0] }
+                  - org_type[1].each do |org|
```
Comment (amcates) [inline app/views/dashboard/protocol_filters/_filter_protocols_form.html.haml]: - options.each do |name, value|

**Comment by jayhardee9** [comment]
Looks good

**Pull request merged**